#235 Use event date instead of publish date for display and recent event list

dedeibel opened the pull request:
Fixing #235

you could argue about sorting of the most recent conferences list on the main page. It would now obey the event date too.

Any comments on that? (FYI @derpeter)

After this would be deployed we could update the conference last released date using a "batch job".

One could argue we should refactor the "event_last_released_at" column. Any comments on that, is it worth it?

derpeter commented:
I agree that the release date is also an important date. We could keep it in the view and add the event date. The current implementation gives the impression that the release date is the event date.

dedeibel commented:
But would you base the "recent conferences" view on the index page on the event date or the release date?

In different words: should imported, old conferences be advertised in the "recent conferences" list?

I have not strong opinion here but I'm tending towards recording date.

derpeter commented:
@dedeibel recent conferences is for me "recent added" maybe we should chance the name?

dedeibel commented:
Updated the PR.

Added release date as additional information if not the same as the event date. Rewording to main page "recently published" -> "recently added" to make the fact more clear.

MaZderMind commented:
nice implementation
